frontend/packages/console-shared/locales/en/console-shared.json

```diff
@@ -141,6 +141,8 @@
   "Provides a list of default categories which are shown in the Developer Catalog. The categories must be added below customization developerCatalog.": "Provides a list of default categories which are shown in the Developer Catalog. The categories must be added below customization developerCatalog.",
   "Add project access roles": "Add project access roles",
   "Provides a list of default roles which are shown in the Project Access. The roles must be added below customization projectAccess.": "Provides a list of default roles which are shown in the Project Access. The roles must be added below customization projectAccess.",
+  "Add page actions": "Add page actions",
+  "Provides a list of all available actions on the Add page in the Developer perspective. The IDs could be used to disable individual actions.": "Provides a list of all available actions on the Add page in the Developer perspective. The IDs could be used to disable individual actions.",
```

Comment:
All the other snippets indicate to the user where in the spec the snippet should go. I think we should include the same here.

Reply:
+1

Reply:
Changed description to

> Provides a list of all available actions on the Add page in the Developer perspective. The IDs must be added below customization addPage disabledActions to hide these actions.

Fine with that?

Reply:
Ya. We really should support markdown in these descriptions ;/